res/controllers/common-controller-scripts.js

```diff
@@ -47,7 +47,7 @@ function stringifyObject(obj, maxdepth, checked, prefix) {
     return obj;
 }
 
-function arrayContains(array, elem) {
+arrayContains = function(array, elem) {
```

Comment:
ES7 has a standard function for this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
I somehow didn't get that to work, I thought we are using some old JS version?

Reply:
Correct, we are. I mentioned that to point out that this function will be redundant with the language built in function once #1795 is merged, so I don't think we should add it now.

Reply:
Well, it is needed for toString in the meanwhile. It's not like it will create any conflicts. But it's a good point, we'll remove it when that PR is merged.